GiEnJul/ClientApp/src/components/verifyConnection/verifyConnection.tsx

```diff
@@ -15,11 +16,12 @@ const VerifyConnection: React.FC<VerifyConnection> = () => {
     useParams<RouteParameters>();
 
   const [error, setError] = useState(false);
+  const apiservice = new ApiService();
 
   useEffect(() => {
-    fetch(`verify/${giverRowKey}/${recipientRowKey}/${partitionKey}`, {
-      method: "POST",
-    }).then((response) => setError(!response.ok));
+    apiservice
+      .post(`verify/${giverRowKey}/${recipientRowKey}/${partitionKey}`, {})
```

Comment:
The data attribute in the post method should probably be set to {} as default. This way it doesn't have to be set in every call that doesn't contain any data.

Reply:
usually when doing  a post, there are some data attached, so it is better to just set it as empty when needed